client/state/selectors/is-site-using-full-site-editing.js

```diff
@@ -0,0 +1,36 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { includes } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import getBlogStickers from 'state/selectors/get-blog-stickers';
+import { hasStaticFrontPage } from 'state/sites/selectors';
+import { getActiveTheme } from 'state/themes/selectors';
+
+/**
+ * List of themes that are supported by Full Site Editing so we can call it "active"
+ *
+ * @type {Array}
+ */
+const supportedThemes = [ 'modern-business' ];
```

Comment:
Hmm, this really shouldn't live on the client and needs a follow up. Maybe an extra prop on the theme response, or a dedicated endpoint? I'm open to ideas.